koans/src/java7/AboutStringsInSwitch.java

```diff
@@ -0,0 +1,27 @@
+package java7;
+
+import com.sandwich.koan.Koan;
+
+import static com.sandwich.koan.constant.KoanConstants.__;
+import static com.sandwich.util.Assert.assertEquals;
+
+public class AboutStringsInSwitch {
+
+    @Koan
+    public void stringsInSwitchStatement () {
+        String[] animals = {"Dog", "Cat", "Tiger", "Elephant", "Zebra"};
+        String dangerous=null;
+        String notDangerous=null;
+        for (String animal : animals){
+            switch (animal) {
+                case "Tiger":
+                        dangerous = animal;
```

Comment:
Oh, right.  I meant a `break` out of the `case`, not a `continue` in the `for`.  Sorry.